source/P0323R9.bs

```diff
@@ -750,7 +750,8 @@ expected& operator=(expected&& rhs) noexcept(see below);
 *Ensures*: `bool(rhs) == bool(*this)`.
 
 *Remarks*: The expression inside noexcept is equivalent to:
-`is_nothrow_move_assignable_v<T>` is `true` and `is_nothrow_move_constructible_v<T>` is `true`.
+`is_nothrow_move_assignable_v<T>` is `true` and `is_nothrow_move_constructible_v<T>` is `true` and
```

Comment:
- May throw while move-constructing during initialization of `val` when `is_nothrow_move_constructible_v<T> ` is false.
- May throw while move-constructing during initialization of `unex` when `is_nothrow_move_constructible_v<E> ` is false.